src/components/PopUpWithConfirm.js

```diff
@@ -0,0 +1,39 @@
+import Popup from "./Popup.js";
+
+export default class PopupWithConfirm extends Popup {
+  constructor(popupSelector, confirmHandler) {
+    super(popupSelector);
+    this._confirmHandler = confirmHandler;
```

Comment:
Needs correction: The name of the function should begin with a verb, reflecting the action that the function performs. Next, the elements to which the action refers are described. `_handleConfirm` would be good.

Reply:
Done